Parse-Dashboard/Authentication.js

```diff
@@ -96,15 +97,17 @@ function authenticate(userToTest, usernameOnly) {
         matchingUsername = user.user;
         // User restricted apps
         appsUserHasAccessTo = user.apps || null;
+        isReadOnly = !!user.readOnly; // make it true/false
       }
 
       return isAuthenticated;
-    }) ? true : false;
+    }) ? true : false ;
```

Comment:
mistake?

Reply:
no because of the && && it was returning an object instead of true / false